sparklemotion/src/main/java/com/ifttt/sparklemotion/Decor.java

```diff
@@ -50,8 +50,8 @@
      */
     final boolean withLayer;
 
-    private Decor(@NonNull View contentView, @NonNull Page page, boolean layoutBehind, Animation slideInAnimation,
-            Animation slideOutAnimation, boolean withLayer) {
+    Decor(@NonNull View contentView, @NonNull Page page, boolean layoutBehind, Animation slideInAnimation,
```

Comment:
It is the correct change for removing synthetic accessor, but the intention is to not be able to instantiate with the constructor. Package private is probably good already here, though.